app/main/views/briefs.py

```diff
@@ -142,21 +144,22 @@ def list_briefs():
             links={},
         )
     else:
-        briefs = briefs.paginate(
+        paginated_briefs = briefs.paginate(
             page=page,
             per_page=current_app.config['DM_API_BRIEFS_PAGE_SIZE'],
         )
 
         return jsonify(
-            briefs=[brief.serialize() for brief in briefs.items],
+            briefs=[brief.serialize(with_users) for brief in paginated_briefs.items],
             meta={
-                "total": briefs.total,
+                "total": paginated_briefs.total,
+                "total_pages": paginated_briefs.pages
```

Comment:
Maybe we should just call this `pages`? 
Seems like it probably means the same thing and it might be more succinct.